helm/values.yaml

```diff
@@ -160,6 +169,14 @@ worker:
     maxReplicas: 100
     targetCPUUtilizationPercentage: 40
     targetMemoryUtilizationPercentage: 80
+    customMetrics: []
```

Comment:
Can we either make this fleet level with an optional fleet agnostic override (similar to how the env vars are done) and then in the template, use a default chain for the priority.

Reply:
this should already be the case, as it's piggybacking off the autoscaling merge